assets/src/edit-story/components/form/dateTime/index.js

```diff
@@ -26,11 +38,7 @@ const DateTimeWrapper = styled.div`
 function DateTime({ value, onChange, is12Hour = true, forwardedRef }) {
   return (
     <DateTimeWrapper ref={forwardedRef}>
-      <DateTimePicker
-        currentTime={value}
-        onChange={onChange}
-        is12Hour={is12Hour}
-      />
+      <TimePicker currentTime={value} onChange={onChange} is12Hour={is12Hour} />
```

Comment:
Note: This part is subject to change once we have style and does not have tests for that reason.

Reply:
Shouldn't this just be a browser default?

Reply:
`is12Hour` currently comes from the WP configuration -- since that value is for the editor only then it seems to make sense to use what's configured by the user.